tests/marqo_test.py

```diff
@@ -180,24 +199,31 @@ def warm_request(self, func, *args, **kwargs):
             func(*args, **kwargs)
 
     def create_cloud_index(self, index_name, settings_dict=None, **kwargs):
+        """
+        Creates a cloud index with the given name and settings. If settings_dict or any kwargs are provided,
+        they will be used to create a unique index name by hashing the settings_dict and kwargs.
+        If settings_dict is not provided, the index name will be the given index_name + the index_suffix.
+
+        Performs check for status of index and waits for index to be ready before returning.
+        """
```

Comment:
nit:
Explain the function params. Especially that kwargs are those that would be passed to a `Index.create()` call